package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "authmosphere",
-  "version": "3.0.1",
+  "version": "3.0.2",
```

Comment:
We could think of adding a comment on the change log, I think we missed than in the previous PR.

Reply:
done